web/apps/labelstudio/src/utils/api-proxy.js

```diff
@@ -460,6 +460,11 @@ export class APIProxy {
           json() {
             return Promise.resolve(response);
           },
+          text() {
+            return JSON.stringify(response);
```

Comment:
If response is a string this method return this string wrapped in quotes which is not what you expect from `text()`